.github/workflows/unit_test.yml

```diff
@@ -77,6 +77,7 @@ jobs:
       uses: codecov/codecov-action@v3
       continue-on-error: true
       with:
+        token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
Codecov has decided (as of, what, today?) that it really really wants a token. I added a secret to the repo as well.